### PR TITLE
Add NDA_LLC_draft_ukraine

### PR DESCRIPTION
**Pull request by freevova**

### CODE REVIEW

File: app/assets/javascripts/contracts.coffee [hunk 1]
```diff
@@ -0,0 +1,18 @@
+$ ->
+  $('.generate_pdf').click (e)->
+    e.preventDefault()
+    $('#contract_pdf').val 'true'
+    $('form').submit()
+  $('.preview').click ->
+    $('#contract_pdf').val 'false'
+
+# Don't show end_date field on NDA templates
+  hide_fields = ['3','4']
```
Comment (EnragedWildkin) [inline app/assets/javascripts/contracts.coffee]: This should not be hard-coded.

File: app/views/contracts/generate/template_1.html.haml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+          до договору № #{@contract.number} від #{locale_date(@contract.start_date, "%d %B %Y", :ua)}
+      %td
+        %p.bold.center Specification of requirements
+        %p.center
+          to contract # #{@contract.number} of #{locale_date(@contract.start_date, "%B %d, %Y", :en)}
+    %tr
+      %td.little_retreat
+        %p
+          Стосовно послуг ІТ Консультанта
+          #{@user.ru_user_field.last_name}а
```
Comment (EnragedWildkin) [inline app/views/contracts/generate/template_1.html.haml]: Need to investigate better solutions for russian/ukrainian inflections.

File: app/views/contracts/generate/template_3.html.haml [hunk 3]
```diff
@@ -0,0 +1,743 @@
+.nda_llc_contract
+  %p.center= "Appendix B to the AGREEMENT # #{@contract.number}"
+  %p.center= "Додаток Б до договору № #{@contract.number}"
+  %br
+  %p.bold.center Положення про захист інтелектуальної власності.
+  - 2.times do
```
Comment (EnragedWildkin) [inline app/views/contracts/generate/template_3.html.haml]: Just two `br`s are ok

**Comment by EnragedWildkin** [comment]
@freevova Please check my comments.

**Comment by freevova** [comment]
@EnragedWildkin Please review.

**Comment by EnragedWildkin** [comment]
@freevova Please rebase

**Pull request closed without merging**